docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md

```diff
@@ -6,6 +6,19 @@ approvers:
 title: kubeadm upgrade
 ---
 {% capture overview %}
+
+`kubeadm upgrade` is a user-friendly command that wraps complex upgrading logic behind one command, with support
+for both planning an upgrade and actually performing it. `kubeadm upgrade` can also be used for downgrading
+cluster if necessary.
+
+Every upgrade process might be a bit different, so we've documented each minor upgrade process individually.
+Please check these documents out for more detailed how-to-upgrade guidance:
+
+ * [1.6 to 1.7 upgrades](/docs/tasks/administer-cluster/kubeadm-upgrade-1-7/)
+ * [1.7.x to 1.7.y  upgrades](/docs/tasks/administer-cluster/kubeadm-upgrade-1-8/)
+ * [1.7 to 1.8 upgrades](/docs/tasks/administer-cluster/kubeadm-upgrade-1-8/)
+ * [1.8.x to 1.8.y upgrades](/docs/tasks/administer-cluster/kubeadm-upgrade-1-8/)
```

Comment:
IT seems that this list doesn't show up in the preview...
What about the 1.8 to 1.9  upgrade?

Reply:
rebase conflict -- please add it if you get to it